src/test/java/com/epam/edumanagementsystem/security/config/AuthenticationTest.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.edumanagementsystem.security.config;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestBuilders.formLogin;
+import static org.springframework.security.test.web.servlet.response.SecurityMockMvcResultMatchers.authenticated;
+import static org.springframework.security.test.web.servlet.response.SecurityMockMvcResultMatchers.unauthenticated;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.redirectedUrl;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class AuthenticationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void testLoggingWithInvalidUser() throws Exception {
+        mockMvc.perform(formLogin()
+                .user("wrong-user@gmail.com").password( "123456"))
```

Comment:
That would be great not to hardcode username and password values, and in general any value, it would be better to have class level Strings.